webpack.config.prod.js

```diff
@@ -4,49 +4,49 @@ const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
-  entry: {
-    helloWorldPage: 'pages/hello-world/helloWorld.jsx',
-  },
-
-  output: {
-    path: path.resolve(__dirname, 'public'),
-    chunkFilename: 'webpack/[chunkhash].js',
-    filename: 'javascripts/[name].js',
-    publicPath: '/assets/',
-  },
+    entry: {
+        helloWorldPage: 'pages/hello-world/helloWorld.jsx',
```

Comment:
Yes, but I don't think we're going to be maintaining a separate webpack config for PROD. It seems inevitable that they're going to get out of sync (and it's already happened). We're going to attempt to merge them into one config and pass through a `dev` flag.

However, it's a fair point, is it worth keeping them in sync while this one still exists?